src/game.cpp

```diff
@@ -8437,10 +8438,18 @@ static void butcher_submenu( const std::vector<map_stack::iterator> &corpses, in
     auto cut_time = [&]( butcher_type bt ) {
         int time_to_cut = 0;
         if( corpse != -1 ) {
-            time_to_cut = butcher_time_to_cut( player_character, *corpses[corpse], bt );
+            const mtype &item = *( *corpses[corpse] ).get_mtype();
```

Comment:
calling a variable `item` hides the `item` class. if the LGTM check doesn't catch it some time i'd be surprised.

Reply:
Oh god, yeah, I shouldn't code at night.